cypress/e2e/spec.cy.js

```diff
@@ -132,4 +132,46 @@ describe('Login Tests', () => {
 
     cy.contains('Password is too short.').should('be.visible')
   })
+
+  it('should display error alert when failed login', () => {
+    cy.visit(Cypress.env('BASE_URL'))
+
+    cy.contains('UnReveal').should('be.visible')
+    cy.contains('Login').should('be.visible')
+    cy.contains('Register').should('be.visible')
+
+    cy.contains('Login').click()
+
+    cy.get('[placeholder="Email"]').type('email@email.com')
+    cy.get('[placeholder="Password"]').type('password')
+
+    cy.intercept(`*login`, {
+      statusCode: 403,
+    })
+
+    cy.contains("Let's go!").click()
+
+    cy.on('windows:alert', (str) => {
+      expect(str).to.equal('Invalid inputs')
+    })
+  })
+
+  it('Login enters main page', () => {
```

Comment:
There is already a definition to a test that should login
https://github.com/WS22-23-SWEPM3ILV-INF-BA-VZ-Group-2/App/blob/4b3ee6416647e5665c984358c13c4f4f445f273e/cypress/e2e/spec.cy.js#L92-L105

You should alter that one and delete this.